src/server/handlers/launchpad.js

```diff
@@ -83,6 +83,14 @@ const RESPONSE_SNAPCRAFT_YAML_NO_NAME = {
   }
 };
 
+const RESPONSE_SNAP_NAME_NOT_REGISTERED = {
+  status: 'error',
+  payload: {
+    code: 'snap-name-not-registered',
+    message: 'Snap name is not registered in the store'
```

Comment:
I'd suggest that we should do this by handling that payload code specially in the frontend component, rather than by injecting HTML from the server-side handler.